src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -253,7 +253,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Overrides Function GetImports() As ImmutableArray(Of String)
             ' TODO: implement (only called from VBI) https://github.com/dotnet/roslyn/issues/5854
-            Throw ExceptionUtilities.Unreachable
+            Dim importNames As New ArrayBuilder(Of String)(GlobalImports.Length)
+            importNames.AddRange(GlobalImports.Select(Function(go) go.Name))
```

Comment:
These issues are my fault--I've never figured out the subtleties of when we prefer Linq and when we avoid it, or when ToImmutableAndFree is appropriate--and apparently I gave Artur bad advice.